ingenialink/eoe/network.py

```diff
@@ -124,7 +124,7 @@ def _build_eoe_command_msg(cmd, node=1, data=None):
         node_field = f"{node:0{EOE_MSG_NODE_SIZE}d}".encode("utf-8")
         null_terminator = b"\x00"
```

Comment:
Do you think that it can be a constant? you use it more than once and in the test

Reply:
Yes, I will create a constant.